tests/pytests/test_json.py

```diff
@@ -293,11 +293,74 @@ def testArrtrim(env):
     # FIXME:
     pass
 
-
 def testAsTag(env):
+    res = env.execute_command('FT.CREATE', 'idx', 'ON', 'JSON',
+                              'SCHEMA', '$.tag', 'AS', 'tag', 'TAG', 'SEPARATOR', ',')
+
+    env.expect('JSON.SET', 'doc:1', '$', '{"tag":"foo,bar,baz"}').ok()
+    env.expect('JSON.SET', 'doc:2', '$', '{"tag":["foo","bar","baz"]}').ok()
```

Comment:
I do not see a test for a combination of an array of strings, each string is coma separated ...

Reply:
Also, this test is wrong which indicates the implementation is wrong. this JSON `{"tag":["foo","bar","baz"]}` should have rais an error with this JSON path `$.tag`, this right JSON path to index it should have been `$.tag[*]`

Reply:
Should `$.tag[*]` be used at create time?